module/VuFind/tests/unit-tests/src/VuFindTest/ServiceManager/Factory/AutowiringFactoryTest.php

```diff
@@ -66,19 +66,19 @@ class AutowiringFactoryTest extends \PHPUnit\Framework\TestCase
     /**
      * Data provider for testAutoWiring
      *
-     * @return array
+     * @return \Iterator<(int | string), mixed>
```

Comment:
I'm not sure why the type annotation is more specific here than elsewhere, but I think we can make it more specific here based on actual usage if we want to -- the key is always a string and the value is always an array.

```suggestion
     * @return \Iterator<string, array>
```